tests/test_cli_edge_cases.py

```
@@ -347,6 +348,79 @@ def test_handle_cli_commands_config_generation_failure(self):
+            # Reset the global config variables
+            import mmrelay.config
+
+            original_base = mmrelay.config.custom_base_dir
+            original_data = mmrelay.config.custom_data_dir
+
+            try:
+                mmrelay.config.custom_base_dir = None
+                mmrelay.config.custom_data_dir = None
+
+                with patch("mmrelay.cli.logger"):
+                    result = handle_cli_commands(args)
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)